mongodbatlas.go

```diff
@@ -180,13 +196,23 @@ func (m *MongoDBAtlas) DeleteUser(ctx context.Context, req dbplugin.DeleteUserRe
 		}
 	}
 
-	// Default to "admin" if no db provided
-	if databaseUser.DatabaseName == "" {
-		databaseUser.DatabaseName = "admin"
+	// If the user is an X.509 user, delete the user from the X.509 certificate subject field
```

Comment:
Suggest to change comment to mention where we're deleting the user from (`$external` db)
```suggestion
	// If the user is an X.509 user, delete the user from the $external database
```